### ISSUE TITLE
Create standard templates for metadata collection

### ISSUE THREAD

**Issue by kimberlyh66** [terraref/tutorials#127]
* Templates are needed for sites and cultivars metadata
* Indicate which fields are required/optional, the data type, and any notes on accepted values or units

**Comment by kimberlyh66** [comment]
Standard template for sites metadata collection should include the following fields:
 
* sitename (required, character)
* city (optional, character)
* state (optional, character)
* country (optional, character)
* mat (optional, integer, unit: degrees Celsius)
* map (optional, integer, unit: mm)
* soil (optional, character)
* notes (optional, character)
* soilnotes (optional, character)
* greenhouse (optional, boolean, values: 'f' or 't')
* geometry (optional, postgis geometry)
* time_zone (optional, character)

**Comment by kimberlyh66** [comment]
Example sites template here: [sites_template.xlsx](https://github.com/terraref/tutorials/files/2907358/sites_template.xlsx)

**Comment by kimberlyh66** [comment]
Template for metadata collection should include the following fields:

* sitename (required, character)
* cultivar_name (required, character)
* specie_id (required, integer, value must exist in `species` table)
* ecotype (optional, character)
* cultivar_notes (optional, character)
* city (optional, character)
* state (optional, character)
* country (optional, character)
* mat (optional, integer, units: degrees Celsius)
* map (optional, integer, units: mm)
* soil (optional, character)
* site_notes (optional, character)
* soilnotes (optional, character)
* greenhouse (optional, boolean, values: 'f' or 't')
* geometry (optional, postgis geometry)
    * will fill column for user if shapefile is provided
* time_zone (optional, character)

**Comment by kimberlyh66** [comment]
An example template can be found here: 
[metadata_collection_template.xlsx](https://github.com/terraref/tutorials/files/2907492/metadata_collection_template.xlsx)

**Comment by dlebauer** [comment]
@kimberlyh66 I think that the goal here is to have a spreadsheet that is standardized and clear what information we will need. Some examples are 

1. one that I've created but isn't very clear ... https://docs.google.com/spreadsheets/d/11WHROTlvsZnMaSoz2cB6091pwtzzulq_P72owpmr8kk/edit#gid=1497549722 but hopefully more simple. 
2. An old version of the genomes2fields data collection template https://docs.google.com/spreadsheets/d/16zeBjLhFGJNSZYnycfjby-sQg63QQ4W07QNf6foLleM/edit#gid=1219992057
3. A MIAPPE compliant ISA-TAB or spreadsheet template (I've requested the spreadsheet template).

The design should be such that it will be straightforward to write functions in R to parse them and insert into the database.

We can discuss further next week.
